fb_instant_articles.module

```diff
@@ -46,24 +116,24 @@ function fb_instant_articles_page_attachments_alter(array &$page) {
  * Implements hook_init().
  */
 function fb_instant_articles_init() {
-  $enable_transformer_logging = \Drupal::config('fb_instant_articles_enable_logging');
-  $appender = array(
+  $enable_transformer_logging = \Drupal::config('fb_instant_articles.base_settings')->get('enable_logging');
+  $appender = [
     'class' => $enable_transformer_logging ? '\Drupal\fb_instant_articles\DrupalLoggerAppender' : 'LoggerAppenderNull',
```

Comment:
Boo, global namespace. Should it be prefixed with a \ then to make it clear, given that the other option is namespaced?